sharktank/tests/layers/resnet_test.py

```diff
@@ -0,0 +1,134 @@
+# Copyright 2024 Advanced Micro Devices, Inc
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import unittest
+
+import torch
+
+from sharktank.models.punet.layers import ResnetBlock2D
```

Comment:
Yeah, we'll want to move this test and the sharding into punet, since this resnet block is just named "resnet" but is more resnet-like for that specific purpose.